package.json

```diff
@@ -3,11 +3,24 @@
   "version": "0.4.10",
   "description": "Polyfill for the dialog element",
   "main": "dialog-polyfill.js",
+  "module": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/GoogleChrome/dialog-polyfill.git"
   },
   "author": "The Chromium Authors",
   "license": "BSD",
-  "homepage": "https://github.com/GoogleChrome/dialog-polyfill"
+  "homepage": "https://github.com/GoogleChrome/dialog-polyfill",
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "mocha": "^5.2.0",
+    "npm-run-all": "^4.1.5",
+    "rollup": "^1.1.2"
+  },
+  "scripts": {
+    "build": "npm-run-all -p build:*",
+    "build:umd": "rollup index.js --file dialog-polyfill.js --format umd --name dialogPolyfill",
+    "prepublishOnly": "npm run build",
```

Comment:
For safer future compatibility, I'd recommend adding:

```js
"build:es": "rollup index.js --file dialog-polyfill.esm.js --format esm",
```

and change `module` to:

```js
"module": "dialog-polyfill.esm.js",
```

By adding a separate bundle for ESM now, this would ensure that if the source ever did its own importing, thereby preventing or deterring ES users from importing it directly, users would not be forced in the future to change paths to point to a new bundle path.